src/module/item/kingmaker-feature/values.ts

```diff
@@ -0,0 +1,17 @@
+import * as R from "remeda";
+import { BehaviorType, KingmakerCategory } from "./types.ts";
+
+const KINGDOM_CATEGORY_DATA = {
+    kingdomFeat: { behavior: "feat" },
+    kingdomFeature: { behavior: "feature" },
+    kingdomActivity: { behavior: "activity" },
```

Comment:
These should be hyphenated since their main use is as values

Reply:
Interestingly enough, feat categories, which this kinda mimics, are lowercase with no hyphens, but I can do hyphens.

Reply:
That's not interesting. They're all single words, so no hyphens would appear.

Reply:
Are you sure?
![image](https://github.com/foundryvtt/pf2e/assets/1286721/c786d421-30f6-4c42-96c8-a21b6367734e)